digicampipe/utils/hist2d.py

```diff
@@ -16,14 +27,115 @@ def fill(self, x, y):
                 range=self.range
             )
             self.histo[pixel_id] += H.astype('u2')
+        self.xedges, self.yedges = xedges, yedges
 
     def contents(self):
         return self.histo
 
+    def fit_y(self, min_entries=2):
```

Comment:
I would not call this fit_y since it is not really a "fit" but computation of mean and standard deviation along a coordinate axis.
Some we can maybe have mean() and std() methods

Reply:
maybe when we move this to cta-sst-1m/histogram (see #253 )